nucypher/cli/characters/alice.py

```diff
@@ -13,59 +10,103 @@
-@click.option('--pay-with', help="Run with a specified account", type=EIP55_CHECKSUM_ADDRESS)
+@click.option('--dev', '-d', help="Enable development mode", is_flag=True)
+@click.option('--force', help="Don't ask for confirmation", is_flag=True)
+@click.option('--dry-run', '-x', help="Execute normally without actually starting the node", is_flag=True)
 @click.option('--teacher-uri', help="An Ursula URI to start learning from (seednode)", type=click.STRING)
 @click.option('--min-stake', help="The minimum stake the teacher must have to be a teacher", type=click.INT, default=0)
-@click.option('--discovery-port', help="The host port to run node discovery services on", type=NETWORK_PORT, default=9151)  # TODO
-@click.option('--http-port', help="The host port to run Moe HTTP services on", type=NETWORK_PORT, default=8151)  # TODO
+@click.option('--discovery-port', help="The host port to run node discovery services on", type=NETWORK_PORT)
+@click.option('--controller-port', help="The host port to run Alice HTTP services on", type=NETWORK_PORT)
 @click.option('--federated-only', '-F', help="Connect only to federated nodes", is_flag=True)
 @click.option('--network', help="Network Domain Name", type=click.STRING)
 @click.option('--config-root', help="Custom configuration directory", type=click.Path())
 @click.option('--config-file', help="Path to configuration file", type=EXISTING_READABLE_FILE)
 @click.option('--provider-uri', help="Blockchain provider's URI", type=click.STRING)
+@click.option('--sync/--no-sync', default=True)
+@click.option('--geth', '-G', help="Run using the built-in geth node", is_flag=True)
+@click.option('--poa', help="Inject POA middleware", is_flag=True, default=None)
 @click.option('--no-registry', help="Skip importing the default contract registry", is_flag=True)
 @click.option('--registry-filepath', help="Custom contract registry filepath", type=EXISTING_READABLE_FILE)
-@click.option('--bob-encrypting-key', help="Bob's encrypting key as a hexideicmal string", type=click.STRING)
-@click.option('--bob-verifying-key', help="Bob's verifying key as a hexideicmal string", type=click.STRING)
+@click.option('--pay-with', help="Run with a specified account", type=EIP55_CHECKSUM_ADDRESS)
+@click.option('--bob-encrypting-key', help="Bob's encrypting key as a hexadecimal string", type=click.STRING)
+@click.option('--bob-verifying-key', help="Bob's verifying key as a hexadecimal string", type=click.STRING)
 @click.option('--label', help="The label for a policy", type=click.STRING)
 @click.option('--m', help="M-Threshold KFrags", type=click.INT)
 @click.option('--n', help="N-Total KFrags", type=click.INT)
-@click.option('--dev', '-d', help="Enable development mode", is_flag=True)
-@click.option('--force', help="Don't ask for confirmation", is_flag=True)
-@click.option('--dry-run', '-x', help="Execute normally without actually starting the node", is_flag=True)
+@click.option('--value', help="Total policy value", type=click.FLOAT)
```

Comment:
```suggestion
@click.option('--value', help="Total policy value (in Wei)", type=click.INT)
```
If you track this value all the way down to the transaction in `BlockchainPolicy` you'll see there's no conversion anywhere. So either you apply this suggestion, or you convert the input to Wei. Either way, the `help` text here must make explicit what denomination it expects.